pkg/server/api/trustdomain/v1/service.go

```diff
@@ -80,7 +80,30 @@ func (s *Service) ListFederationRelationships(ctx context.Context, req *trustdom
 }
 
 func (s *Service) GetFederationRelationship(ctx context.Context, req *trustdomainv1.GetFederationRelationshipRequest) (*types.FederationRelationship, error) {
-	return nil, status.Error(codes.Unimplemented, "unimplemented")
+	log := rpccontext.Logger(ctx)
```

Comment:
can you add:
```
	rpccontext.AddRPCAuditFields(ctx, logrus.Fields{telemetry.TrustDomainID: req.TrustDomain)
```
It will add that field to audit logs, (in case of error or success